requirements.in

```diff
@@ -2,6 +2,8 @@
 # See LICENSE file for licensing details.
 
 envoy_data_plane==0.2.5
+# from observability_libs.v0.juju_topology.py
+cosl
```

Comment:
I'm not sure what has changed to cause the tests to fail without it, do you have an idea?
If this is necessary now in the requirements for any charm that uses the `juju_topology` lib, then we might expect the CI to fail for many of our charms. If that's the case, I suggest you file an issue about this so that we can address it.

Reply:
Yeah the `cosl` dependency is required because the newest version of `juju_topology` requires `cosl`.  I'll bring this up in our daily today to discuss what to do